Change Jenkins Template to OCP v4.1 based Template

makentenza opened the pull request:
Move existing Jenkins Persistent template to OCP v4.1 based Template

oybed commented on templates/jenkins.yml:
```diff
@@ -1,8 +1,8 @@
-apiVersion: v1
+apiVersion: template.openshift.io/v1
```
BTW: this file is also missing the `---`

makentenza commented:
Thanks for reviewing this, just added the missing characters. Re: strategy for merging this, yes, let's sync on this topic, I have added further info in the other PR #28

oybed commented:
Per convo - please open a new one that targets a `-v4.1` branch rather than the main one (as this will cause breakage)
